host/xtest/xtest_main.c

```diff
@@ -150,6 +154,12 @@ int main(int argc, char *argv[])
 
 	opterr = 0;
 
+	if (argc > 1 && !strcmp(argv[1], "sha-perf")){
```

Comment:
Well that would not add great complexity, and it would save some typing. But I can also create wrapper scripts instead of symlinks so not a big deal.